package com.desafioitau.api.transferencia.v1.notificacao.exception;

public class NotificacaoServiceUnavailableException extends NotificacaoException {

    public NotificacaoServiceUnavailableException(Throwable cause) {
        super(cause);
    }
}
